src/index.js

```diff
@@ -1,17 +1,42 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {
+  BrowserRouter as Router, Switch, Route, Link,
+} from 'react-router-dom';
+import { Provider } from 'react-redux';
+import store from './redux/configureStore';
+import Books from './components/books';
+import Categories from './components/categories';
+
 import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+
+function App() {
+  return (
+    <Router>
+      <div id="navlinks">
+        <div className="nav-link">
+          <Link to="/">Books</Link>
+        </div>
+        <div className="nav-link">
+          <Link to="/categories">Categories</Link>
+        </div>
+      </div>
+
+      <Switch>
+        <Route exact path="/">
+          <Books />
+        </Route>
+        <Route path="/categories">
+          <Categories />
+        </Route>
+      </Switch>
+    </Router>
+  );
+}
```

Comment:
- [ ] This seems to be duplicate code, please check the implementation you did in the `app.js` looks the same as this one, please implement index.js to keep the only app.js after wrapping it inside the `provider store`